### PR TITLE
Simplified readme

### LINKED ISSUE #123: Simplify and refactor README
We should greatly simplify the README file to make installation steps clearer. We should factor out instructions that are not immediately relevant to the most common installation flow.

image

### PR DESCRIPTION
**Pull request by justinpobrien**
Updated README and added a few other docs. This PR closes #123

### CODE REVIEW

File: README.md
Comment (jamaljsr) [inline README.md:38]: I don't think this info needs to be right here in the main README. It can get moved to the config docs for remote & integrated modes.

File: README.md
Comment (jamaljsr) [inline README.md:66]: @guggero can correct me if I'm wrong but I believe `v0.2.0-alpha` requires `lnd` `v0.11.1-beta`

File: doc/remote.md
Comment (jamaljsr) [inline doc/remote.md:11]: Only the `admin.macaroon` file is needed. They do not need to copy over all of the macaroons.

File: doc/remote.md
Comment (jamaljsr) [inline doc/remote.md:1]: Why have two docs on configuring the remote mode? Shouldn't this content just live in the `config-remote-lnd.md` file?

File: README.md
Comment (jamaljsr) [inline README.md:2]: You removed the Troubleshooting section but didn't move it to another doc. You don't think this information is necessary? If you don't want to display it directly in the readme, I think we should have a separate doc with the content that it linked to from the readme.

**Review by jamaljsr — CHANGES_REQUESTED** [review]
Overall, the updated organization of the docs is definitely better. I left some feedback inline.

File: doc/remote.md
Comment (justinpobrien) [reply doc/remote.md:11]: I had assumed I just needed tls.cert and admin.macaroon but when I actually went to run it I was prompted for the other macaroons. I'll do more digging

File: doc/remote.md
Comment (justinpobrien) [reply doc/remote.md:1]: So yes and no. I had originally leaned towards one combined file but then realized that for the happy path a user would only need a subset of the information. Perhaps later we trim and merge the two?

File: README.md
Comment (justinpobrien) [reply README.md:2]: SGTM

File: README.md
Comment (justinpobrien) [reply README.md]: Good point. Changed to make them next to each other

File: doc/remote.md
Comment (justinpobrien) [reply doc/remote.md:1]: Ok combined into one and added the quickstart section

File: doc/config-lnd-remote.md
Comment (jamaljsr) [inline doc/config-lnd-remote.md:42]: Should we change this to "Additional Configuration" or something like that? This subtitle seems redundant with the title of the doc above.

**Review by jamaljsr — COMMENTED** [review]
A couple more comments after your latest updates.

File: doc/config-lnd-remote.md
Comment (justinpobrien) [reply doc/config-lnd-remote.md:42]:
```suggestion
## Additional Configuration
```

File: README.md
Comment (justinpobrien) [reply README.md]: Yeah these are probably not critical to have to be honest. I've removed them to keep things simple

**Review by jamaljsr — APPROVED** [review]

**Comment by jamaljsr** [comment]
LGTM 🥇

File: README.md
Comment (guggero) [inline README.md:22]: Should we change this to `--uipassword=UP48lm4Vjqxy<change_this_or_you_will_get_robbed>` similar to how `bitcoind` shows default passwords in their docs?

File: README.md
Comment (guggero) [inline README.md:27]: Add "... variable to avoid it being recorded in the command history"?

File: README.md
Comment (guggero) [reply README.md:60]: On the code/dependency level, yes. But on the RPC level, to connect to a remote node, `v0.11.0-beta` is correct (this isn't even the requirement of LiT itself but faraday).

File: README.md
Comment (guggero) [inline README.md:19]: In my parts of the documentation I always use the `lnd` notation (lower case and with back ticks to mark as inline code). Not sure what the "preferred" notation is but we should probably try to use the same notation everywhere.

File: README.md
Comment (guggero) [inline README.md:43]: I think the part about removing the `lnd.` prefix in the `lnd.conf` will probably confuse most users. Maybe this doesn't need to be mentioned at all?

File: doc/compile.md
Comment (guggero) [inline doc/compile.md:11]: nit: Capitalize sentence and add full stop, same for `protoc`.

File: doc/remote.md
Comment (guggero) [reply doc/remote.md:11]: Yeah, loop and faraday need all macaroons, unfortunately.

**Review by guggero — COMMENTED** [review]
Nice job splitting this into multiple files! The README itself looks much more approachable now and not give the impression that this is very complicated.
Looks mostly good to me, only found some small things.

**Comment by guggero** [comment]
Can be rebased against master now that the dependent PR is in.

**Comment by jamaljsr** [comment]
Rebased on top of master. 

Waiting on @justinpobrien's responses to @guggero's review before merging.

File: doc/compile.md
Comment (justinpobrien) [reply doc/compile.md]: Fixed

File: README.md
Comment (justinpobrien) [reply README.md]: Good point. Changed up wording a bit

File: README.md
Comment (justinpobrien) [reply README.md]: Fixed

File: README.md
Comment (justinpobrien) [reply README.md]: Fixed

File: README.md
Comment (justinpobrien) [reply README.md]: Good idea. Fixed

**Review by guggero — APPROVED** [review]
Very nice, LGTM :100:

**Pull request merged**
